src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -54,7 +55,7 @@ public class ArmSubsystem extends SubsystemBase{
     //spark absolute encoder
     private SparkAbsoluteEncoder m_absEncoder;  
     //embedded relative encoder
-    //private RelativeEncoder m_Encoder;
+    private RelativeEncoder m_encoder;
```

Comment:
Remove this encoder. The AbsoluteEncoder is now wired directly into the sparkmax which replaces the RelativeEncoder entirely.